tests/conftest.py

```diff
@@ -35,7 +35,6 @@ def pytest_collection_modifyitems(items) -> None:
     """Applies tags to tests based on folders that they are in."""
     top_test_dir = Path(__file__).parent
     top_doctest_dir = top_test_dir.parent.joinpath("src/snowflake/snowpark")
-    modin_doctest_dir = top_doctest_dir.joinpath("modin/pandas")
```

Comment:
we missed those doctests under new plugin folders

Reply:
Are there any files in the new plugin folders that need to have doctests run? All the docstring overrides get applied to the appropriate class in modin/pandas before doctests for those are run.

Reply:
oh, so they did run? One change we will have in index is that we will only have the index plugin but no index.py in the modin frontend at least in our repo. So we have to include them to run.

One more question, once we use modin OSS, how to run doctests?

Reply:
> oh, so they did run? One change we will have in index is that we will only have the index plugin but no index.py in the modin frontend at least i
n our repo. So we have to include them to run.

Sure, that makes sense.

> One more question, once we use modin OSS, how to run doctests?

This is something I'm not too sure about, maybe @sfc-gh-dpetersohn would know. My guess is we'd have to mess with conftest.py a little to run the doctests defined in the docstring overrides, but I'm not sure how to handle docstrings that we don't override (if any).

Reply:
track it here https://snowflakecomputing.atlassian.net/browse/SNOW-1440221